kodi_addon_checker/check_repo.py

```diff
@@ -1,21 +1,14 @@
 import os
 import sys
-import json
+
 import kodi_addon_checker.check_addon as check_addon
 from kodi_addon_checker.common import colorPrint
+from kodi_addon_checker.report import Report, INFORMATION, Record, ReportManager
 
 
-def _read_config_for_version(repo_path):
-    config_path = os.path.join(repo_path, '.tests-config.json')
-    if os.path.isfile(config_path):
-        with open(config_path) as json_data:
-            return json.load(json_data)
-
-    return None
-
-
-def check_repo(repo_path, parameters):
-    error_counter = {"warnings": 0, "problems": 0}
+def check_repo(config, repo_path, parameters):
+    repo_report = Report(repo_path)
+    repo_report.add(Record(INFORMATION, "Checking repository %s" % repo_path))
     print("Repo path " + repo_path)
```

Comment:
I think we can remove this line, as it's logged above

Reply:
Fixed